frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.scss

```diff
@@ -0,0 +1,29 @@
+.kubevirt-create-vm-modal__virtual-hardware-tab-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+
+  .kubevirt-create-vm-modal__virtual-hardware-tab-cd-title {
+    margin-top: var(--pf-global--spacer--md);
+  }
+
+  .kubevirt-create-vm-modal__virtual-hardware-tab-empty-state {
+    flex-direction: column;
+    align-items: start;
+    margin-top: var(--pf-global--spacer--sm);
+    padding-left: var(--pf-global--spacer--md);
+
+    .pf-c-title {
+      color: var(--pf-c-wizard__nav-link--m-disabled--Color);
+    }
+  }
+}
+
+.kubevirt-create-vm-modal__virtual-hardware-tab-main {
+  flex: 1;
+}
+
+.kubevirt-create-vm-modal__virtual-hardware-tab-boot-select {
```

Comment:
boot select?

Reply:
If we keep these we should swap them out for PF4 spacers. I don't think we have a boot select here anymore so this is maybe just old code that can be removed?